src/Microsoft.Health.Dicom.Web/Startup.cs

```diff
@@ -52,17 +55,20 @@ public virtual void Configure(IApplicationBuilder app)
     }
 
     /// <summary>
-    /// Adds ApplicationInsights for telemetry and logging.
+    /// Adds Open telemetry exporter for Azure monitor.
     /// </summary>
-    private void AddApplicationInsightsTelemetry(IServiceCollection services)
+    private void AddOpenTelemetry(IServiceCollection services)
     {
         string instrumentationKey = Configuration["ApplicationInsights:InstrumentationKey"];
 
         if (!string.IsNullOrWhiteSpace(instrumentationKey))
         {
             var connectionString = $"InstrumentationKey={instrumentationKey}";
 
-            services.AddApplicationInsightsTelemetry(aiOptions => aiOptions.ConnectionString = connectionString);
+            services.AddSingleton(Sdk.CreateMeterProviderBuilder()
+                .AddMeter("Microsoft.Health.Dicom.*")
```

Comment:
Should `Microsoft.Health.Dicom` be defined somewhere so that meters can use it as a base?